### DISCUSSION TITLE
Glide cache - images errors

### DISCUSSION THREAD

**Discussion by ConstantMath** [area17/twill#1370] [General]
Hello, best wishes everybody here !
I have a question concerning Glide taht must be relatd to the cache system & config files.
I made some changes to the .env & rebuild the laravel cache and all my images disapear.

It was working before.
Is there a way to rebuild the Glide cache files ?

**Comment by ifox** [comment]
Hi @ConstantMath sorry for the delay, but I think you're also on discord so assuming you figured this out!

The Glide cache is dynamically created from http requests to your images with specific parameters.
You can certainly warm that cache if you know all your variations programmatically by issuing requests to the Glide controller, but you can also let the cache recreate itself.

Can you clarify what you mean by all my images disappear? The content of the Glide cache folder? It should be totally fine to drop the cache folder as your image urls in your frontend (and twill's frontend itself) should be pointing to the dynamic /img path, not the generated cache files.

**Reply by ConstantMath** [reply]
Hello @ifox , thanks for your reply. 
If I remember correctly it was indeed a cache problem, related to Laravel then.
I don't know why but on some projects when I empty the cache, I have to empty the routes cache a second time otherwise the media routes are broken.

Anyway, I now systematically use this command :
`php artisan optimize && php artisan route:clear && php artisan config:clear`

Best,
